docs/variants/protectli_vp2420/building-manual.md

```diff
@@ -33,8 +35,15 @@ VP2420.
     git submodule update --init --checkout
     ```
 
-1. Obtain the Protectli blobs package, extract it and copy just `protectli`
-   directory to `3rdparty/blobs/mainboard` directory.
+1. Obtain the Protectli blobs package by downloading the zip file from the
+   [newest tag](https://github.com/Dasharo/protectli-blobs/tags).
+1. Extract it from coreboot repository to the `3rdparty/blobs/mainboard`
+   directory and customize the folder name:
+
+    ```bash
+    unzip protectli_blobs.zip -d 3rdparty/blobs/mainboard
+    mv 3rdparty/blobs/mainboard/protectli-blobs-1.0.18 3rdparty/blobs/mainboard/protectli
```

Comment:
The newest tag will not always be 1.0.18. Also the tags do not match the v1.0.0 nor v1.1.0 versions, we have to do something about it first...

Reply:
I universalized these commands

Reply:
Great, but it does not fix the actual problem entirely. vp2420 does not have v1.0.18 version for example, so how one does know what tag to use for it? Also VP4670 will have v1.1.0 like vp2420 does, so the tags will have to change probably to

protectli_vault_ehl_v1.1.0, protectli_vault_cml_v1.1.0, protectli_vault_cml_v1.0.18, etc.

Reply:
Currently, the documentation says to simply download the zip from the latest tag, the version of which does not have to be equal to the firmware version(but probably should). From what I understand, at this point, we have one blob package for all Protectli's models, and we want to separate it so that each family has separate tags. In that case, changes are required on the protectli-blobs repository and only then can I adjust the documentation accordingly.